utils/mock.js

```diff
@@ -37,29 +35,30 @@ function mockQuery (schema, mockedSchema, parsedQuery, fixture, saveFixture, glo
       // Check if the query receives args and check if the required ones are passed
       argumentsValidator(queryArgs, querySchema[0].arguments, name, queryVariables)
       // If there are fixtures, set the values
-      if (fixture && fixture.data) {
+      if (fixture && fixture.data !== undefined) {
         mock = setFixture(mock, fixture.data, name)
         if (saveFixture) {
           mockedSchema[Query][name] = mock
         }
       }
       // Return the mock of the selected fields
-      mockedQuery = { [queryName]: mock }
-      if (globalQueryVariables) {
-        return {
-          globalQueryVariables: queryVariables,
-          mockedQuery
-        }
-      }
-      return mockedQuery
+      return Object.assign(
```

Comment:
Is there a way to refactor this block of code, is not that easy to read? also, some comments will be great!

Shall we create a function that returns it? we are using the same code multiple times :)